text/10_what_is_software_testing.md

```diff
@@ -42,9 +42,9 @@ Keep in mind that I am someone who teaches a class on software testing.  Even I
 
 ## Verification and Validation
 
-Software testing also involves ensuring that the right software was created.  Imagine the following conversation between a tester and a customer:
+Software testing also involves ensuring that the right software was created.  Imagine the following conversation between a project manager and a customer:
```

Comment:
Occasionally, depending on the project, but you're right, a project manager is __much__ more likely to do so.